run

```diff
@@ -37,11 +37,19 @@ def main():
     if not args.skip_build:
         build_swift_toolchain(workspace, args)
 
+    clang_flag = []
+    if args.clang_source_path or args.clangc:
+        clangc = args.clangc
+        if args.clang_source_path:
+            clangc = build_clang(workspace, args.clang_source_path)
+        if clangc:
+            clang_flag = ['--clangc', clangc]
```

Comment:
I think `clang_flags` (plural) would be better as this could contain multiple flags. Same in `execute_runner` and `execute_build_incremental` below.

Reply:
Thanks! I renamed it to `clang_flags`.

Reply:
I thought we could just use
```
common-check_execute([ninja_path])
```
and avoid looking up `ninja` through `xcrun` again, but of course that doesn’t set the SDK environment variables etc.